process/factory/shard/vmContainerFactory_test.go

```diff
@@ -66,3 +89,89 @@ func TestVmContainerFactory_Create(t *testing.T) {
 	acc := vmf.VMAccountsDB()
 	assert.NotNil(t, acc)
 }
+
+func makeDummyGasSchedule() map[string]uint64 {
+	gasSchedule := make(map[string]uint64)
+
+	gasSchedule = arwenconfig.FillGasMapWithWASMOpcodeValues(gasSchedule, 1)
+	gasSchedule["StorePerByte"] = 1
```

Comment:
move api calls gas filler to a public function in arwen.

Reply:
Done.